src/gui/wallet.go

```diff
@@ -76,6 +76,11 @@ func walletSpendHandler(gateway Gatewayer) http.HandlerFunc {
 			return
 		}
 
+		if err := r.ParseForm(); err != nil {
```

Comment:
Isn't ParseForm called automatically?

Reply:
Oh, it's called automatically, don't know why it didn't work yesterday, maybe I forgot to restart the node. I will remove this.